cross/mono/PLIST

```diff
@@ -1,94 +1,73 @@
-rsc:bin/al
-rsc:bin/al2
-rsc:bin/csc
-rsc:bin/csc-dim
-rsc:bin/csi
-rsc:bin/gacutil
-rsc:bin/gacutil2
-rsc:bin/ilasm
-rsc:bin/makecert
-rsc:bin/mcs
-rsc:bin/mdassembler
-rsc:bin/mdoc-assemble
-rsc:bin/mdoc-export-html
-rsc:bin/mdoc-export-msxdoc
-rsc:bin/mdoc-update
-rsc:bin/mdoc-validate
-rsc:bin/mdvalidater
-rsc:bin/mkbundle
+bin:bin/al
+bin:bin/al2
+bin:bin/csc
+bin:bin/csi
+bin:bin/gacutil
+bin:bin/gacutil2
+bin:bin/ilasm
+bin:bin/makecert
+bin:bin/mcs
+bin:bin/mdassembler
+bin:bin/mdoc-assemble
+bin:bin/mdoc-export-html
+bin:bin/mdoc-export-msxdoc
+bin:bin/mdoc-update
+bin:bin/mdoc-validate
+bin:bin/mdvalidater
+bin:bin/mkbundle
 bin:bin/mono
 bin:bin/mono-boehm
-rsc:bin/mono-configuration-crypto
-rsc:bin/monodis
-rsc:bin/monodocer
-rsc:bin/monodocs2html
-rsc:bin/monodocs2slashdoc
-rsc:bin/mono-find-provides
-rsc:bin/mono-find-requires
-rsc:bin/mono-gdb.py
-rsc:bin/mono-heapviz
-rsc:bin/monop
-rsc:bin/monop2
-rsc:bin/mono-package-runtime
-rsc:bin/mono-service
-rsc:bin/mono-service2
+bin:bin/mono-configuration-crypto
+bin:bin/mono-find-provides
+bin:bin/mono-find-requires
+bin:bin/mono-gdb.py
+bin:bin/mono-heapviz
+bin:bin/mono-package-runtime
+bin:bin/mono-service
+bin:bin/mono-service2
 bin:bin/mono-sgen
-rsc:bin/mono-sgen-gdb.py
-rsc:bin/mono-test-install
+bin:bin/mono-sgen-gdb.py
+bin:bin/mono-test-install
+bin:bin/monodis
+bin:bin/monodocer
+bin:bin/monodocs2html
+bin:bin/monodocs2slashdoc
+bin:bin/monop
+bin:bin/monop2
 bin:bin/mprof-report
-rsc:bin/nunit-console
-rsc:bin/nunit-console2
-rsc:bin/nunit-console4
-rsc:bin/peverify
-rsc:bin/resgen
-rsc:bin/resgen2
-rsc:bin/sn
-rsc:bin/vbc
-rsc:bin/wsdl
-rsc:bin/wsdl2
-rsc:etc/mono/2.0/Browsers/Compat.browser
-rsc:etc/mono/2.0/DefaultWsdlHelpGenerator.aspx
-rsc:etc/mono/2.0/machine.config
-rsc:etc/mono/2.0/settings.map
-rsc:etc/mono/2.0/web.config
-rsc:etc/mono/4.0/Browsers/Compat.browser
-rsc:etc/mono/4.0/DefaultWsdlHelpGenerator.aspx
-rsc:etc/mono/4.0/machine.config
-rsc:etc/mono/4.0/settings.map
-rsc:etc/mono/4.0/web.config
-rsc:etc/mono/4.5/Browsers/Compat.browser
-rsc:etc/mono/4.5/DefaultWsdlHelpGenerator.aspx
-rsc:etc/mono/4.5/machine.config
-rsc:etc/mono/4.5/settings.map
-rsc:etc/mono/4.5/web.config
-rsc:etc/mono/browscap.ini
-rsc:etc/mono/config
+bin:bin/peverify
+bin:bin/resgen
+bin:bin/resgen2
+bin:bin/sn
+bin:bin/vbc
+bin:bin/wsdl
+bin:bin/wsdl2
+lib:lib/libMonoPosixHelper.so
+lib:lib/libMonoSupportW.so
 lib:lib/libikvm-native.so
-lnk:lib/libmono-2.0.so
-lnk:lib/libmono-2.0.so.1
+lib:lib/libmono-2.0.so
+lib:lib/libmono-2.0.so.1
```

Comment:
please keep `lnk` here (and others in this PLIST) as those are links!

Reply:
indeed, fixed : )